optin/puller/puller/db/CompletedOptinDatabase.py

```diff
@@ -39,7 +39,12 @@ def insert_mapping(self, nem_address_dict, symbol_address_dict):
 		nem_balance = reduce(lambda x, y: x + y, nem_address_dict.values())
 		symbol_balance = reduce(lambda x, y: x + y, symbol_address_dict.values())
 
-		if nem_balance != symbol_balance:
+		ngl_source = [
+			'NATRUSTUAB5LAWDSOWDUPQUFYRTGZTJZDGML2JKP',
+			'NANODESTSN7GU76QPLGMJ7BCGCAA2PHVBZZUUI62',
+			'NBKXLNQ2GKTLQBTXNFOHULNJDA7T2Q57CZGQ2TFP'
+		]
+		if nem_balance != symbol_balance and set(ngl_source) != set(nem_address_dict.keys()):
```

Comment:
can we just modify json instead of this hack ? 😅

Reply:
not sure, but I think it's rather make more sense to have it visible in the code, could additionally throw in amounts if you want

Reply:
i'd rather not special case this 😅